pkg/defaults/policies/files/severity_high_privileged.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "b1df1abb-e5a5-4ff7-98fe-1d28a22b55d8",
   "name": "Privileged Containers with Important and Critical Fixable CVEs",
-  "description": "Alert on containers running in privileged mode with important or critical fixable vulnerabilities",
```

Comment:
We use the wording `with important or critical` here and below we use `a Severity Rating at least Important`, which should be the same, right? I think we should use the same wording for both to emphasize that the only difference is the `privileged` part. Also the titles `Privileged Containers with Important and Critical Fixable CVEs` and `Fixable Severity at least Important` seem unnecessarily different wording wise.

Reply:
good call.